lib/src/test/java/tech/relaycorp/awaladroid/storage/persistence/DiskPersistenceTest.kt

```diff
@@ -14,16 +11,14 @@ import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
 import org.junit.Test
-import org.junit.runner.RunWith
 
-@RunWith(AndroidJUnit4::class)
 internal class DiskPersistenceTest {
 
-    private val context: Context = ApplicationProvider.getApplicationContext()
     private val coroutineScope = TestCoroutineScope()
+    private val filesDir = File.createTempFile("rootDir", "droid_test").parent ?: "/temp"
```

Comment:
Actually, I misread [the docs](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#getParent())... The API docs says the return type is `String` but the description says it could be `null`.

But still, `File.createTempFile(prefix, suffix)` couldn't create a file at the root of the filesystem, so there would always be a parent.